### DISCUSSION TITLE
Token Server 是否开源

### DISCUSSION THREAD

**Discussion by xiaopao2014** [wechaty/wechaty#2086] [Q&A] [answered]
🚨 The issue tracker is not for questions 🚨

If you have a question, please ask it on https://stackoverflow.com/questions/tagged/wechaty

[question]
是否可以搭建自己的Token server

**Comment by huan** [comment, marked as answer]
Of course, you can!

Please refer to #1986 and make you a [Wechaty Puppet Service Provider](https://github.com/wechaty/puppet-services/issues/1)!
